.github/workflows/int-test-azure-workflow.yml

```diff
@@ -0,0 +1,191 @@
+name: Integration Test on Azure WorkFlow
+on:
+  push:
+    branches:
+      - develop
+      - master
+      - CSPL-1946
```

Comment:
just putting it here so that we don't forget t o remove this branch name before merging.